Integrations/PaloAltoNetworksCortex/PaloAltoNetworksCortex.py

```diff
@@ -665,7 +1079,87 @@ def search_by_file_hash_command():
         'ReadableContentsFormat': formats['markdown'],
         'HumanReadable': tableToMarkdown('Logs ' + table_name + ' table', screened_results),
         'EntryContext': {
-            'Cortex.Logging(val.id==obj.id)': output
+            'Cortex.Logging(val.id === obj.id)': output
+        }
+    }
+    return entry
+
+
+def query_table_logs_command(command_data_dict: dict):
+
+    args = demisto.args()
+
+    start_time = args.get('startTime')
+    end_time = args.get('endTime')
+    limit = args.get('logsAmount', '5')
+    time_range = args.get('timeRange')
+    time_value = args.get('rangeValue')
+
+    if time_range:
+        if time_value:
+            service_end_date = datetime.now()
+            service_start_date = get_start_time(time_range, int(time_value))
+        else:
+            raise Exception('Enter timeRange and timeValue, or startTime and endTime')
+    else:
+        # parses user input to datetime object
+        service_start_date = datetime.strptime(start_time, "%Y-%m-%d %H:%M:%S")
```

Comment:
dont use `strptime` and `strftime`, use either `datetime` or `dateutil` builtin functions